silverback/application.py

```diff
@@ -22,6 +23,24 @@ class TaskData:
     handler: AsyncTaskiqDecoratedTask
 
 
+class SharedState(defaultdict):
+    def __init__(self):
+        # Any unknown key returns None
+        super().__init__(lambda: None)
+
+    def __getattr__(self, attr):
+        try:
+            super().__getattr__(attr)
+        except AttributeError:
+            return super().__getitem__(attr)
```

Comment:
question: was this sole return intended?

Reply:
good find